feature/policy_forwarding/encapsulation/otg_tests/encap_gre_ipv4/README.md

```diff
@@ -0,0 +1,88 @@
+# PF-1.2: Policy-based traffic GRE Encapsulation to IPv4 GRE tunnel
+
+## Summary
+
+The test verifies policy forwarding(PF) encapsulation action to IPv4 GRE tunnel when matching on source/destination.
+
+## Setup
+
+
+*   DUT has an ingress port and 2 egress ports.
+
+    ```
+                             |         | --eBGP-- | ATE Port 2 |
+        [ ATE Port 1 ] ----  |   DUT   |          |            |
+                             |         | --eBGP-- | ATE Port 3 |
+    ```
+
+*   Traffic is generated from ATE Port 1.
+*   ATE Port 2 is used as the destination port for encapsulated 
+    traffic.
+*   ATE Port 3 is used as the fallback destination for
+    pass-through traffic.
+
+### Configuration
+
+1.  All DUT Ports are configured as a singleton IP interfaces.
+
+2.  IPv4 and IPv6 static routes to test destinations networks IPV4-DST/IPV6-DST are configured on DUT towards ATE Port 3.
+
+3.  Static routes to 32x IPv4 GRE encap destinations towards ATE Port 2.
```

Comment:
Do you mean another sets of IPv4 and IPv6 static routes  than the one in step 2 pointing to 32x IPv4 GRE tunnel destinations? If yes, then can you please specify?